packages/share_plus/share_plus_platform_interface/lib/method_channel/method_channel_share.dart

```diff
@@ -117,7 +119,8 @@ class MethodChannelShare extends SharePlatform {
   /// then make new file in TemporaryDirectory and return with path
   /// the system will automatically delete files in this
   /// TemporaryDirectory as disk space is needed elsewhere on the device
-  Future<XFile> _getFile(XFile file, {String? tempRoot}) async {
+  Future<XFile> _getFile(XFile file,
+      {String? tempRoot, String? nameOverride}) async {
```

Comment:
```suggestion
  Future<XFile> _getFile(
    XFile file, {
    String? tempRoot, 
    String? nameOverride,
  }) async {
```

Adding a comma and formatting the method